docs/guide/web-animation.md

```diff
@@ -37,16 +37,20 @@ Please refer to [this section](web-scene.md#how-to-embed-a-web-scene-in-your-web
 
 The web animation is played by a web component from the [WebotsAnimation.js] package called `webots-animation`.
 
-To play the animation automatically, the `playWhenReady` attribute of the web component has to be set to `true` before the loading of the page.
-
-By default, the name of the X3D and JSON files will be deduced from the HTTP URL, but the `title` attribute of the web component can be used to define a custom name.
+The following attributes are available:
+* `x3d`: the name of the .x3d file containing the animation world.
+* `json`: the name of the .json file containing the animation steps.
```

Comment:
```suggestion
* `json`: the name of the .json file containing the animation sequence.
```
That seems more appropriate, see for example https://docs.unrealengine.com/4.27/en-US/AnimatingObjects/SkeletalMeshAnimation/Sequences/